src/coffee/sync/utils/product.coffee

```diff
@@ -432,13 +425,12 @@ class ProductUtils extends BaseUtils
 
           else if _.has(image, 'label') and
           (image.label.length == 1 or image.label.length == 2)
-
-            actions.push(
+            action =
               action: 'changeImageLabel'
```

Comment:
I wonder if it is some outdated action not supported by API anymore as I can see only `setImageLabel` in the doc.

Reply:
https://github.com/sphereio/sphere-node-sdk/pull/324/commits/e8462920f582e80ee06a138b90d91c59762a80f7